### ISSUE TITLE
Onetime exception on multi project build when publishing test dependancies

### ISSUE THREAD

**Issue by DomBlack** [Sensatus/sbt-build-info-conf#2]
```
java.lang.UnsupportedOperationException: tail of empty list
    at scala.collection.immutable.Nil$.tail(List.scala:339)
    at scala.collection.immutable.Nil$.tail(List.scala:334)
    at scala.reflect.internal.SymbolTable.popPhase(SymbolTable.scala:172)
    at scala.reflect.internal.Symbols$Symbol.unsafeTypeParams(Symbols.scala:1477)
    at scala.reflect.internal.Symbols$TypeSymbol.tpe(Symbols.scala:2777)
    at scala.reflect.internal.Mirrors$Roots$RootPackage$.<init>(Mirrors.scala:268)
    at scala.reflect.internal.Mirrors$Roots.RootPackage$lzycompute(Mirrors.scala:267)
    at scala.reflect.internal.Mirrors$Roots.RootPackage(Mirrors.scala:267)
    at scala.reflect.internal.Mirrors$Roots.RootPackage(Mirrors.scala:245)
    at scala.reflect.internal.Mirrors$RootsBase.init(Mirrors.scala:238)
    at scala.reflect.runtime.JavaMirrors$class.createMirror(JavaMirrors.scala:34)
    at scala.reflect.runtime.JavaMirrors$class.runtimeMirror(JavaMirrors.scala:61)
    at scala.reflect.runtime.JavaUniverse.runtimeMirror(JavaUniverse.scala:12)
    at scala.reflect.runtime.JavaUniverse.runtimeMirror(JavaUniverse.scala:12)
    at com.sensatus.SbtBuildInfoConf$.generateResource(SbtBuildInfoConf.scala:235)
    at com.sensatus.SbtBuildInfoConf$$anonfun$projectSettings$1.apply(SbtBuildInfoConf.scala:173)
    at com.sensatus.SbtBuildInfoConf$$anonfun$projectSettings$1.apply(SbtBuildInfoConf.scala:172)
    at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
    at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
    at sbt.std.Transform$$anon$4.work(System.scala:63)
    at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:226)
    at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:226)
    at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
    at sbt.Execute.work(Execute.scala:235)
    at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:226)
    at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:226)
    at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
    at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
    at java.util.concurrent.FutureTask.run(FutureTask.java:266)
    at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
    at java.util.concurrent.FutureTask.run(FutureTask.java:266)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
    at java.lang.Thread.run(Thread.java:745)
```

**Comment by analytically** [comment]
+1

**Comment by MaxWorgan** [comment]
Fixed in release 1.1.0
